tsconfig.json

```diff
@@ -6,9 +6,7 @@
     "noEmitOnError": true,
     "baseUrl": ".",
     "paths": {
-      "ember": ["node_modules/at-types-ember"],
-      "npm:*": ["local-types/*"],
-      "*": ["local-types/*"]
+      // @types all included implicitly
```

Comment:
@poteto TRUE! This is what happens when you leave a note for yourself. Thanks for catching it.